ui/app/styles/core/buttons.scss

```diff
@@ -386,3 +386,17 @@ a.button.disabled {
     }
   }
 }
+
+.hds-button {
```

Comment:
Doing the same thing as we did for copy button, making new classes to match structure. When we remove finally `.button` class we can audit these with design

Reply:
Ok so this is to preserve the existing styling but ultimately we will want consistent HDS styling once the button component is adopted and this can be removed? If so a comment might be helpful just so it's easy to spot when it comes time to remove.

Reply:
yep - there's a comment for this section above the `.hds-copy-button` class changes. I can add another here